src/i18n/it_IT/vehicles.ts

```diff
@@ -606,4 +606,310 @@ export default {
+    },
+    63: {
+        caption: 'Camion Movimento Terra',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [0, 18],
+    },
+    64: {
+        caption: 'AV/CA Unità Cinofila SAR',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [0, 18],
+    },
+    65: {
+        caption: 'AF Unità Cinofila SAR',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [0, 18, 30],
+    },
+    66: {
+        caption: 'ACT/Scar PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    67: {
+        caption: 'Modulo Scarrabile Movimento Terra',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    68: {
+        caption: 'Modulo Scarrabile HCP',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    69: {
+        caption: 'AF/BUS PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    70: {
+        caption: 'CA/POLI',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    71: {
+        caption: 'AF Unità Cinofila SAR',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    72: {
+        caption: 'Funzionario AIB - DOS PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    73: {
+        caption: 'Pickup AIB - CA/ESK PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    74: {
+        caption: 'Autocarro AIB - AF/BOSC PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
+    75: {
+        caption: 'Autobotte AIB PC',
+        color: '#0a2bff',
+        credits: -1,
+        coins: -1,
+        staff: { min: 1, max: 2 },
+        icon: 'car-side',
+        possibleBuildings: [30],
+    },
```

Comment:
trainings for staff missing here probably, but they can also be added later in theory.

Reply:
I know but at this moment I don't know what the training programs are for the aforementioned vehicles